cmd/kogito/command/test/common.go

```diff
@@ -51,6 +61,7 @@ func SetupCliTest(cli string, factory context.CommandFactory, kubeObjects ...run
 	kogitoRootCmd.Command().SetArgs(strings.Split(cli, " "))
 	kogitoRootCmd.Command().SetOut(testOut)
 	kogitoRootCmd.Command().SetErr(testErr)
+	kogitoRootCmd.Command().ParseFlags([]string{"--config", GetTestConfigFilePath()})
```

Comment:
Nice that this is working. I would've join this args to the `cli` command `string` instead. Don't need to change, just an observation. =)

Actually is a better solution since the test could've send `--config` args and here we are overriding it.

Reply:
updated

Reply:
Oh my bad, I meant that **your solution** is better! Please keep yours! I'm sorry, reading the comment again is really open for misinterpretation.

Reply:
ok, I will set back the original ;)

Reply:
done